src/Task.ts

```diff
@@ -862,11 +862,37 @@ export class Task {
         )})`;
     }
 
+    /*
+     * Escape a string so it can be used as part of a RegExp literally.
+     * Taken from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping
+     */
+    private escapeRegExp(s: string) {
+        return s.replace(/([.*+?^=!:${}()|[]\/\\])/g, '\\$1');
+    }
+
+    /*
```

Comment:
Trivia, but the other function docs in the code begin with `/**` - two `*`.

WebStorm does show the function description fine, but consistency is useful, I feel.

Reply:
Fixed.